From bff65db081ee1ee5df0a42231ebf8e8fbb27cccf Mon Sep 17 00:00:00 2001
From: Jens Georg <mail@jensge.org>
Date: Mon, 31 Jul 2023 13:56:08 +0200
Subject: [PATCH] build: Temporarily disable -Werror=deprecated

There is an issue replacing the SAX document validation with DOM
validation
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index f4b629f..5d08a42 100644
--- a/meson.build
+++ b/meson.build
@@ -71,7 +71,7 @@ endif
 
 common_cflags = cc.get_supported_arguments(test_cflags)
 
-add_project_arguments(cc.get_supported_arguments('-Werror=deprecated-declarations'), language: 'c')
+#add_project_arguments(cc.get_supported_arguments('-Werror=deprecated-declarations'), language: 'c')
 conf = configuration_data()
 conf.set_quoted('DATADIR', join_paths(get_option('prefix'), get_option('datadir'), 'gupnp-av'))
 conf.set_quoted('VERSION', meson.project_version())
-- 
2.43.0

